ai/integrations/autogen/pyproject.toml

```diff
@@ -9,7 +9,8 @@ readme = "README.md"
 license = { text="Apache-2.0" }
 requires-python = ">=3.9"
 dependencies = [
-    "autogen-agentchat~=0.2",
+    "autogen-agentchat>=0.4.0",
```

Comment:
Seems like 0.4.2 is the latest version (0.4.0 is yanked)

Reply:
And I think I know why after working on a tutorial... the RoundRobinGroupChat for 0.4.0 was completely broken and would stall if more than 2 tools were submitted with the GroupChatManager not knowing how to pass a function call request for execution.